src/main/java/florizz/core/Parser.java

```diff
@@ -48,10 +97,56 @@ private static FlowerCommand handleFlowerCommand(String input) {
         return new FlowerCommand(occasion);
     }
 
+    private static AddFlowerCommand handleAddFlower(String argument) throws FlorizzException {
+        if (argument == null) {
+            throw new FlorizzException("No argument detected! Please use the correct format of 'add <flowerName> /q <quantity> /to <bouquetName>");
+        }
+
+        if (!argument.matches(ADD_FLOWER_REGEX)) {
```

Comment:
woah i like this idea! will see if i can implement this for my own commands

Reply:
Thanks! That's great 😄